crates/continuations/src/lib.rs

```diff
@@ -22,9 +22,9 @@ pub mod types {
     /// Types used by `Payloads` struct
     pub mod payloads {
         /// type of length
-        pub type Length = usize;
+        pub type Length = u32;
         /// type of capacity
-        pub type Capacity = usize;
+        pub type Capacity = u32;
         /// Type of the entries in the actual buffer
         pub type DataEntries = u128;
     }
```

Comment:
I am not really a fan of these type aliases. I don't really think their utility is worth the extra mental indirection.